src/app/dynamic-forms/fields/link-multichoice/link-multichoice.component.ts

```diff
@@ -245,6 +259,16 @@ export class LinkMultichoiceComponent implements OnChanges, OnDestroy, OnInit {
   }
 
 
+  /**
+   * The function used to determine whether or not dropdown options represent the same entity
+   */
+  compareFn(a: DropdownOption, b: DropdownOption): boolean {
+
+    // Adding the `false` part ensures that the comparison will still return a boolean if either value is nullish
+    return (a?.id === b?.id) || false;
+  }
```

Comment:
Would it be better to cast to boolean with `!!`? So like `return !!(a?.id === b?.id);`

Reply:
I wish we could rely on coerceBooleanProperty, but seeing as we can't this is probably a more consistent syntax. I'll change it.

Reply:
Apparently neither are necessary. Ran some tests and `===` always returns a boolean.